app/assets/javascripts/router.js

```diff
@@ -121,16 +121,34 @@ class ReactRouter extends React.Component<Props> {
                 path="/"
                 render={() =>
                   isAuthenticated ? (
-                    <DashboardView userId={null} isAdminView={false} />
+                    <DashboardView userId={null} isAdminView={false} initialTabKey={null} />
                   ) : (
                     <SpotlightView />
                   )
                 }
               />
+              <SecuredRoute
+                isAuthenticated={isAuthenticated}
+                path="/dashboard/:tab"
+                render={({ match }: ContextRouter) => {
+                  const tab: ?string = match.params.tab;
+                  const initialTabKey: ?string = tab ? urlTokenToTabKeyMap[tab] : null;
+                  return (
+                    <DashboardView
+                      userId={null}
+                      isAdminView={false}
+                      initialTabKey={initialTabKey || null}
```

Comment:
I think the `|| null` part can be removed?

Reply:
Correct, didn't know that `undefined || "default"` results in `"default"` :slightly_smiling_face: